src/app/factories/__tests__/google-analytics.factory.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { StatusCodes } from 'http-status-codes'
```

Comment:
same comment as on the factory - should we move these into the frontend module tests?

Reply:
thanks, shifted accordingly